contracts/proxy/Clones.sol

```diff
@@ -17,6 +17,16 @@ pragma solidity ^0.8.19;
  * _Available since v3.4._
  */
 library Clones {
+    /**
+     * @dev A `CREATE` clone instance deployment failed.
+     */
+    error ERC1167FailedCreateClone();
+
+    /**
+     * @dev A `CREATE2` clone instance deployment failed.
+     */
+    error ERC1167FailedCreate2Clone();
```

Comment:
I would merge these into one error. I realize that the existing revert reasons where different, but I don't see where the additional bit of information (whether it was create or create2) would become relevant.

Reply:
Now that you mention it, the ERC1167 doesn't even mention `CREATE` or `CREATE2` so I merged them. I would even consider something more appealing to the EIP, maybe `ERC1167FailedMinimalProxyDeployment`, what do you think?

Reply:
That's too long I think.

Reply:
What do you think of an alternative closer to the corresponding EIP terminology?

Reply:
The term "clone" appears in the EIP, so I don't think it's a bad term to use.